README.md

```diff
@@ -195,4 +195,4 @@ If you like this project please give it a star
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- Hi, Microverse recommends using [MIT](https://github.com/microverseinc/readme-template/blob/master/LICENSE) licensed. Therefore I would advise you to follow this requirement because it is very professional to add a license to your project.